webapp/client/src/pages/PersonBHasDisabilityPage.js

```diff
@@ -0,0 +1,75 @@
+import PropTypes from "prop-types";
```

Comment:
Is the PersonBHasDisabilityPage any different from the one for Person A? Feels to me like we should be able to have 99% of the code implemented by a single component which gets rendered with a prop or two for the header, field name etc. on the two pages.

Reply:
Yeah. I had it in one page at the beginning. The problem is the backend, because the fields refer 1:1 to the frontend.

Example:
In the frontend I had a property: "hasDisability" instead of "personAHasDisability" & "personBHasDisability" and tried in the backend to rename the field before sending it to the UI and also then changed the name of the field "hasDisability" for the post correspond to "personAHasDisability" or "personBHasDisability". Everything was shiny and nice but while testing I noticed that internally I made another mapping to the form fields (routes.py -> post step) at the end of the story I looked at others implementation and implemented it exactly the same way.

Reply:
Alright, so Julian and I might have had and solved the same problem in #507. Let's not change it for this page in this PR, instead I'll add a sub-ticket to fix that afterwards.